src/midiEditorCommands.cpp

```diff
@@ -1260,6 +1260,22 @@ void cmdMidiInsertNote(Command* command) {
 	outputMessage(s);
 }
 
+void cmdMidiPasteEvents(Command* command) {
+	HWND editor = MIDIEditor_GetActive();
+	MediaItem_Take* take = MIDIEditor_GetTake(editor);
+	int oldCount = MIDI_CountEvts(take, nullptr, nullptr, nullptr);
+	MIDIEditor_OnCommand(editor, command->gaccel.accel.cmd);
+	int newCount = MIDI_CountEvts(take, nullptr, nullptr, nullptr);
+	int count = 0;
+	if (newCount > oldCount) {
```

Comment:
This will mean we report "0 events added" when pasting nothing. I wonder whether we should report "nothing pasted" here like we do for cmdPaste for consistency. That also simplifies things a bit:

```
int added = newCount - oldCount;
if (added <= 0) {
	outputMessage(translate("nothing pasted"));
	return;
}
```